tomviz/python/Recon_DFT.py

```diff
@@ -33,17 +31,12 @@ def transform_scalars(self, dataset):
         self.progress.message = 'Initialization'
         Nz = Ny
         w = np.zeros((Nx, Ny, Nz // 2 + 1)) #store weighting factors
-        v = pyfftw.n_byte_align_empty(
-            (Nx, Ny, Nz // 2 + 1), 16, dtype='complex128')
-        v = np.zeros(v.shape) + 1j * np.zeros(v.shape)
-        recon = pyfftw.n_byte_align_empty(
-            (Nx, Ny, Nz), 16, dtype='float64', order='F')
-        recon_fftw_object = pyfftw.FFTW(
-            v, recon, direction='FFTW_BACKWARD', axes=(0, 1, 2))
+        v = pyfftw.empty_aligned(
+            (Nx, Ny, Nz // 2 + 1), dtype='complex64', n=16)
```

Comment:
16 bit aligned is likely our best choice to take advantage of SSE in modern 64 bit processors, and should likely remain our default choice unless we get fancier in querying for the best alignment.